addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,221 @@
+                    else:
+                        previous_price = float_round(
+                            (previous_price * previous_qty + move["price_unit"] * move["product_qty"]) / total_qty,
+                            precision_digits=precision_price)
+                    svl_vals = _prepare_in_svl_vals(
+                        move, move["product_qty"], move["price_unit"], product, is_dropship)
+                    svl_in_vals_list.append(svl_vals)
+                    previous_qty = total_qty
+                # Add out svl
+                if move["move_type"] == "out" or is_dropship:
+                    qty = move["product_qty"]
+                    if product.cost_method in("average", "fifo") and not is_dropship:
+                        # Reduce remaininig qty in svl of type "in"
+                        while qty > 0 and svl_in_index < len(svl_in_vals_list):
+                            if svl_in_vals_list[svl_in_index]["remaining_qty"] >= qty:
+                                candidate_cost = (svl_in_vals_list[svl_in_index]["remaining_value"] /
+                                                  svl_in_vals_list[svl_in_index]["remaining_qty"])
+                                svl_in_vals_list[svl_in_index]["remaining_qty"] -= qty
+                                svl_in_vals_list[svl_in_index]["remaining_value"] = float_round(
+                                    candidate_cost * svl_in_vals_list[svl_in_index]["remaining_qty"],
+                                    precision_digits=precision_price)
+                                qty = 0
+                            else:
+                                qty -= svl_in_vals_list[svl_in_index]["remaining_qty"]
+                                svl_in_vals_list[svl_in_index]["remaining_qty"] = 0.0
+                                svl_in_vals_list[svl_in_index]["remaining_value"] = 0.0
+                                svl_in_index += 1
+                    svl_vals = _prepare_out_svl_vals(
+                        move, move["product_qty"], previous_price, product)
+                    svl_out_vals_list.append(svl_vals)
+                    previous_qty -= move["product_qty"]
+                # Add manual adjusts after last move
+                if product.cost_method in ("average", "fifo") and not float_is_zero(
+                        previous_qty, precision_digits=precision_uom):
+                    while h_index < len(history_lines):
+                        price_history_rec = history_lines[h_index]
+                        if float_compare(price_history_rec["cost"], previous_price, precision_digits=precision_price):
+                            svl_vals = _prepare_man_svl_vals(
+                                price_history_rec, previous_price, previous_qty, company, product)
+                            svl_man_vals_list.append(svl_vals)
+                            previous_price = price_history_rec["cost"]
+                        h_index += 1
```

Comment:
The first that I found, it should be out the loop:
```suggestion
            # Add manual adjusts after last move
            if product.cost_method in ("average", "fifo") and not float_is_zero(
                    previous_qty, precision_digits=precision_uom):
                while h_index < len(history_lines):
                    price_history_rec = history_lines[h_index]
                    if float_compare(price_history_rec["cost"], previous_price, precision_digits=precision_price):
                        svl_vals = _prepare_man_svl_vals(
                            price_history_rec, previous_price, previous_qty, company, product)
                        svl_man_vals_list.append(svl_vals)
                        previous_price = price_history_rec["cost"]
                    h_index += 1
```

Reply:
Of course, that was the idea, that tab was added by mistake at some point. Sorry